static/templates/authentication/login.html

```diff
@@ -1,28 +1,31 @@
-<div  layout="row" flex="50" layout-align="center center">
-  <md-content class="authentication-form" layout-padding class="md-gray-theme" flex="50" layout-align="center center" layout="column">
-    <h3 id="form-header">Sign In to Crowd Research</h3>
-    <form name="loginForm">
-      <div layout= layout-sm="column">
-        <md-input-container>
-          <label>Email or Username</label>
-          <input required ng-model="login.username">
-          <div ng-if="login.username==null" ng-messages="loginForm.description.$error">
-            <div ng-message="required">This is required.</div>
+<md-dialog>
+    <md-dialog-content>
+        <div  layout="row" flex="50" layout-align="center center">
+          <div class="authentication-form" layout-padding class="md-gray-theme" flex="100" layout-align="center center" layout="column">
+            <h3 id="form-header">Sign In to Crowd Research</h3>
+            <form name="loginForm">
+              <div layout= layout-sm="column">
+                <md-input-container>
+                  <label>Email or Username</label>
+                  <input required ng-model="login.username">
+                  <div ng-if="login.username==null" ng-messages="loginForm.description.$error">
+                    <div ng-message="required">This is required.</div>
+                  </div>
+                </md-input-container>
+              </div>
+              <div layout="row">
+                <md-input-container>
+                  <label>Password</label>
+                  <input type="password" required ng-model="login.password">
+                  <div ng-if="login.password==null" ng-messages="loginForm.description.$error">
+                    <div ng-message="required">This is required.</div>
+                  </div>
+                </md-input-container>
+              </div>
+              <div class="authentication-error" layout="row" ng-show="login.error && loginForm.$pristine" ng-bind="login.error"></div>
+              <md-button ng-click="login.login()" class="md-raised">Sign In</md-button>
+              </form>
           </div>
-        </md-input-container>
-      </div>
-      <div layout="row">
-        <md-input-container>
-          <label>Password</label>
-          <input type="password" required ng-model="login.password">
-          <div ng-if="login.password==null" ng-messages="loginForm.description.$error">
-            <div ng-message="required">This is required.</div>
-          </div>
-        </md-input-container>
-      </div>
-      <div class="authentication-error" layout="row" ng-show="login.error && loginForm.$pristine" ng-bind="login.error"></div>
-      <md-button ng-click="login.login()" class="md-raised">Sign In</md-button>
-      </form>
-  </md-content>
-</div>
-
+        </div>
+    </md-dialog-content>
```

Comment:
in case this is going to be a dialog the borders and shadows have to be removed, same for registration
i would wait on this